src/include/duckdb.h

```diff
@@ -29,11 +33,15 @@
 //! duplicate of duckdb/main/winapi.hpp
 #ifndef DUCKDB_EXTENSION_API
 #ifdef _WIN32
+#ifdef DUCKDB_STATIC
+#define DUCKDB_API
```

Comment:
shouldn't this be `DUCKDB_EXTENSION_API`?

Reply:
Oh, yes, thanks for catching this!